config/mongodb.go

```diff
@@ -0,0 +1,64 @@
+package config
+
+import (
+	"fmt"
+	"os"
+
+	"gopkg.in/yaml.v3"
+)
+
+// collections names
+const (
+	TykDatabase = "tyk_analytics"
+
+	PortalCatalogue     = "portal_catalogue"
+	PortalConfiguration = "portal_configuration"
+	PortalCSS           = "portal_css"
+	PortalMenus         = "portal_menus"
+
+	TykAnalytics           = "tyk_analytics"
```

Comment:
why do you have two constants with same value?

Reply:
![image](https://user-images.githubusercontent.com/70487540/209540552-3aa3eca4-0dc7-4ca7-bcde-558a007410f4.png)
first value its a name of database, second value its the name of collaction

Reply:
Then how about to rename them to TykAnalyticsDB and TykAnalyticsCollection? Or create a struct?

Reply:
fixed